source/index.html.md

```diff
@@ -981,6 +1167,11 @@ POST https://gateway.clearhaus.com/credits
   </dd>
 </dl>
 
+<p class="alert alert-info">
+<b>Notice:</b> Implicitly, <code>initiator</code> is <code>merchant</code> and
+<code>credential_on_file</code> is <code>use</code>.
+</p>
+
```

Comment:
Aah, for a capture?

Reply:
Fro a credit ... or ... am I misunderstanding something?

Reply:
Right! This is placed under Credits. I'm sorry about the noise.